src/GC/GC.jl

```diff
@@ -3,77 +3,135 @@
 
 Garbage collection of Python objects.
 
-See `disable` and `enable`.
+See [`enable`](@ref), [`disable`](@ref) and [`gc`](@ref).
```

Comment:
I'm assuming these can be removed from the module docstring since no longer in use?